test/extended/apiserver/resource_quota.go

```diff
@@ -0,0 +1,61 @@
+package apiserver
+
+import (
+	"context"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-api-machinery][Feature:APIServer][Serial]", func() {
```

Comment:
One nit, please move this entire contents of this file to https://github.com/openshift/origin/tree/master/test/extended/quota/resourcequota.go because we already have other resource quota tests there. Actually it's enough if you move just the `g.It(...` block contents.

Reply:
Updated